src/ng-select/ng-select.component.html

```diff
@@ -21,11 +21,20 @@
                 [ngTemplateOutletContext]="{ items: selectedValues, clear: clearItem }">
         </ng-template>
 
-        <div *ngIf="showFilter()" class="ng-input">
+        <div *ngIf="!isDisabled" class="ng-input">
+            <div *ngIf="!searchable"
+                tabindex="0"
+                (focus)="onInputFocus()"
```

Comment:
maybe it is good idea to rename this methods `onInputFocus` to be more general, like `onFocus`, `onBlur`?